consumer/dragonboard/dragonboard845c/installation/board-recovery.md

```diff
@@ -65,8 +65,10 @@ UFS provisioning is not support yet.
 
 If the flashing process succeeded, all the right bootloaders and partition table should have been set. And fastboot can now be used to flash Linux root file system. The first thing to try is to get into fastboot, to make sure the flashing completed properly.
 
-* Power off the board and make sure no USB cable is plugged into the board
-* Power on the device using the proper power supply
+* Disconnect the power cable from the board and make sure no USB cable is plugged into the board
+* Hold down the "VOL-" button while reconnecting the power supply.
```

Comment:
@johnstultz-work I am following a bit different steps to get the board into fastboot. Can you try the following:

- Disconnect the power cable from the board and make sure no USB cable is plugged into the board
- Connect the power supply
- Press and hold VOL-
- While holding VOL- press and hold ON/OF until the blue LED turns on
- Release ON/OFF once the blue LED is on and release VOL- a couple of seconds after

Reply:
@ndechesne Both AmitP and I have seen similar situations after doing QDL flashing where due to whatever is on the UFS storage the bootloader thinks it is valid to try to boot linux, and it doesn't go into fastboot mode as expected.

I'd much rather have clear instructions that work all the time, rather then shorter instructions that fail often and need fallback documentation.

Reply:
@sgermanserrano Your instructions look ok as well. Though I feel like I've had some cases where on plugging the board in it powers on and doesn't wait for the ON/OFF button. That's why my instructions suggest holding "VOL-" before applying power.

Reply:
@johnstultz-work , obviously I, too, want the right documentation. I was asking before we have different status, so i wanted to understand the difference. thanks for the clarifications, i guess the problem is that when we flash we QDL we don't reset the boot* partitions. so if you had a boot image flashed, and run QDL again and if the ptable didn't change, then the boot partition would still contain the previous image. I guess that's a valid bug that we need to fix. I will add a fake/zero'd boot image on the bootloader package then.

Reply:
hmm. alternatively we could create a custom set of rawprogram XML files that would flash the whole system at once. e.g. we could write the boot, system, super, persist, files which are the output of the AOSP builds. so that the process of flashing the board would be one step , e.g. run QDL and no need to run fastboot.

Reply:
well, about my previous comment, i realized that qdl doesn't support sparse images, so flashing sparse images is not possible for now.. though it could be a good improvement to qdl at some point..

@johnstultz-work , i have added an empty/zero boot image which should now be flashed with qdl and default rawprogram xml files. can you please check/confirm if it fixes the issue you reported?

Reply:
@ndechesne  Personally, I'd rather not try to pack more stuff into QDL, and just keep to the standard AOSP fastboot process.

As for the zero bootimage, I'm probably ok with that as a solution (haven't yet been able to validate), but I still think having the instructions on how to manually enter fastboot is useful to have well documented.  Making it part of the normal instructions makes sure everyone learns it.

Reply:
ok, fair enough. well at least now i think we understand why we had different behavior in the first place, which was bugging me.